micrometer-observation/src/main/java/io/micrometer/observation/docs/DocumentedObservation.java

```diff
@@ -187,6 +188,30 @@ default <T extends Observation.Context> Observation observation(@Nullable Observ
         return observation;
     }
 
+    /**
+     * Creates an {@link Observation} for the given {@link ObservationConvention}. You
+     * need to manually start it. The {@link Observation.Context} creation happens after
+     * the {@link ObservationRegistry} type check. When the {@link ObservationRegistry} is
+     * a no-op, this method fast returns a no-op {@link Observation} and skips the
+     * creation of the {@link Observation.Context}.
+     * @param customConvention convention that (if not {@code null}) will override any
+     * pre-configured conventions
+     * @param defaultConvention default convention that will be picked if there was
+     * neither custom convention nor a pre-configured one via
+     * {@link ObservationRegistry.ObservationConfig#observationConvention(GlobalObservationConvention[])}
+     * @param contextSupplier observation context supplier
+     * @param registry observation registry
+     * @return observation
+     */
+    default <T extends Observation.Context> Observation createNotStarted(
```

Comment:
I think we should call these methods `createNotStarted` but I'm not sure we all agreed on that.

I think either we call all of them `createNotStarted` or `observation`.